dashboard/app/models/user.rb

```diff
@@ -417,15 +417,11 @@ def user_level_for(script_level)
                         level_id: script_level.level_id)
   end
 
-  def levels_from_script(script, stage = nil)
+  def levels_from_script(script)
     ul_map = user_levels_by_level(script)
-    q = script.script_levels.includes(:level, :script, :stage).order(:position)
+    sls = script.script_levels.sort { |a, b| a.position <=> b.position }
```

Comment:
if you do need to do this it's a bit more compact to say `script.script_levels.sort_by(&:position)` (if you want to get in to implementation details, sort_by is different from sort in that it makes a pass to transform the objects into the sort values, which in this case might be a minuscule performance hit, see the doc for Enumerable#sort_by)